sfja/src/components/Administration/DataExports.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable max-len */
+import React from 'react';
+import {Cookies, withCookies} from 'react-cookie';
+import apiUrl from '../../utils/Env';
+import {instanceOf} from 'prop-types';
+import Paper from '@material-ui/core/Paper';
+import {Button} from '@material-ui/core';
+import {saveAs} from 'file-saver';
+
+// eslint-disable-next-line require-jsdoc
+class DataExports extends React.Component {
+  static propTypes = {
+    cookies: instanceOf(Cookies).isRequired,
+  };
+
+  // eslint-disable-next-line require-jsdoc
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  // eslint-disable-next-line require-jsdoc
+  downloadData(toDownload) {
+    const {cookies} = this.props;
+    const body = {
+      toDownload: toDownload,
+    };
+    fetch(apiUrl() + '/dataDownload', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': `Bearer ${cookies.get('token')}`,
+      },
+      body: JSON.stringify(body),
+    }).then((response) => (response.blob()))
+        .then((blob) => {
```

Comment:
@jchin01 you can then use `saveAs` from the `file-saver` library to download a specified file